src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -44,7 +44,8 @@ overload.methods.declaration=Overload methods should not be split. Previous over
 parameter.assignment=Assignment of parameter ''{0}'' is not allowed.
 require.this.method=Method call to ''{0}'' needs \"{1}this.\".
 require.this.variable=Reference to instance variable ''{0}'' needs \"{1}this.\".
-return.count=Return count is {0,number,integer} (max allowed is {1,number,integer}).
+return.count=Return count is {0,number,integer} (max allowed for non-void methods/lambdas is {1,number,integer}).
+return.countVoid=Return count is {0,number,integer} (max allowed for void methods/constructors/lambdas methods is {1,number,integer}).
```

Comment:
`for void methods/constructors/lambdas methods is` - word `methods` is repeated twice, please fix here and in all translations.

Reply:
done. In all further translations it is not repeated.